packages/server/src/api/queueRoutes.ts

```diff
@@ -137,3 +118,43 @@ export const queueRoutes: ServerRoute[] = [
     },
   },
 ];
+
+// for some reason, JOI.allow(null) means the property has to exist, but can be null
+
+async function questionModelToQuestion(qm: QuestionModel): Promise<Question> {
+  return {
+    creator: await userModelToUserPartial(await (await qm.creator).user),
+    id: qm.id,
+    createdAt: qm.createdAt,
+    status: qm.status,
+    text: qm.text,
+    // qm.taHelped: types says is nonnullable, but it is nullable
```

Comment:
> qm.taHelped: typescript says it is nonnullable, but it is nullable

TS says property exists with correct type (nonnullable), but it is nullable (for example, run the tests). this is a mistake with the types we have currently defined, imo, because it makes sense that `taHelped` might not exist.

Reply:
imo this should be changed asap. seems like a mistake.

Reply:
update: will and I talked about this, and it is already nullable, typescript just is weird.